odoo/addons/base/ir/ir_ui_view.py

```diff
@@ -153,6 +153,7 @@ def remove_element(node):
 xpath_utils['hasclass'] = _hasclass
 
 TRANSLATED_ATTRS_RE = re.compile(r"@(%s)\b" % "|".join(TRANSLATED_ATTRS))
+WRONGCLASS = re.compile(r"(@class\s*=|=\s*@class|contains\(@class)")
```

Comment:
> In fact, this might be an interesting way to allow developers to suppress the warning and still get the intended = effect.

But do we want that? There is no situation in which `@class` should be checked using `=`, that simply does not fit the logical semantics of that attribute.

> Maybe we should remove the warning for contains entirely, as the chances of errors are a lot lower.

I don't think they're any lower, at best it's even more misleading and subtly prone to doing the wrong thing.